registry/keyring.go

```diff
@@ -27,6 +27,7 @@ type KeyringSection interface {
 	GetKeyring() envelope.KeyringInf
 	KeyringVersion() int
 	FindMember(*identity.ID) (*primitive.KeyringMember, *primitive.MEKShare, error)
+	FindMemberByKeyID(*identity.ID) (*primitive.KeyringMember, *primitive.MEKShare, error)
```

Comment:
This is really just looking for the `MEKShare` for a specific user, perhaps rename to `FindMEKShare` and make it explicit that it only accepts a PublicKeyID.

Furthermore, it should error if it can't find the MEKShare for the given ID or if the user's don't match? Or perhaps this is the wrong area to do that checking (the mekshare isn't sent from the server anyhow).